draftjs_exporter/dom.py

```diff
@@ -37,23 +37,24 @@ def camel_to_dash(camel_cased_str):
         return dashed_case_str.replace('--', '-')
 
     @classmethod
-    def use(cls, engine=DOM_HTML5LIB):
+    def use(cls, engine=HTML5LIB):
         """
         Choose which DOM implementation to use.
         """
         if engine:
```

Comment:
Not sure what this `if engine` is doing here since there is a default value. Could be removed.

Reply:
Well technically someone could still pass `None` here, although I don't see the point of doing that.

If `engine` is `None`, it won't currently do anything (and leave the `DOM` instance in an invalid state because `cls.DOM` isn't initialised) so it should either raise `ValueError` or not do the check at all and let it crash further down the line. I'd vote for the later.

Reply:
On another note, it seems that `cls.dom` is essential for the instance to function well, therefore, unless we have a really good reason to delay it, this initialisation should be moved to the `__init__` function.